demo/agilebank/templates/k8srequiredprobes_template.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: templates.gatekeeper.sh/v1beta1
+kind: ConstraintTemplate
+metadata:
+  name: k8srequiredprobes
+spec:
+  crd:
+    spec:
+      names:
+        kind: K8sRequiredProbes
+      validation:
+        openAPIV3Schema:
+          properties:
+            message:
+              type: string
+
+  targets:
+    - target: admission.k8s.gatekeeper.sh
+      rego: |
+        package k8srequiredprobes
+
+        get_constraint_message(parameters) = msg {
+          msg := parameters.message
+        }
+
+        violation[{"msg": msg}] {
+          probe_violation[{"msg": msg}]
+        }
+
+        probe_violation[{"msg": msg}] {
+          container := input.review.object.spec.template.spec.containers[_]
```

Comment:
This is not looking at `Pod`. what if users deployed pod directly?

Reply:
Thanks. Noted.